pdns/mbedtlscompat.hh

```diff
@@ -65,10 +69,12 @@ typedef md_type_t mbedtls_md_type_t;
 
 #define mbedtls_entropy_init entropy_init
 #define mbedtls_entropy_func entropy_func
+#define mbedtls_entropy_free entropy_free
 
-#define mbedtls_ctr_drbg_init
+#define mbedtls_ctr_drbg_init ctr_drbg_init
 #define mbedtls_ctr_drbg_seed ctr_drbg_init
```

Comment:
Are you sure about this change? mbedtls_ctr_drbg_init() is always flowed by a  mbedtls_ctr_drbg_seed() in the code. And the argument count of mbedtls_ctr_drbg_init and ctr_drbg_init do not match